front/app/components/ProjectAndFolderCards/components/ProjectsTabPanel.tsx

```diff
@@ -40,11 +40,11 @@ const ProjectsTabPanel = ({
 }: Props) => {
   return (
     <Container
-      id={getTabPanelId(currentTab)}
+      id={getTabPanelId(tab)}
       role="tabpanel"
-      className="e2e-projects-list"
+      className={`e2e-projects-list ${tab === currentTab ? 'active-tab' : ''}`}
```

Comment:
I suppose there was indeed a tiny bug?

Reply:
This class was just added for the e2e tests: because of a11y reasons we sort of have to render all the content in all the tabs, which is why cypress detects 18 project cards instead of 6. This tells it to only look in the active/visible tab.

The bug was actually in the `id` and `aria-labelledby`, where we were giving all tabs the same of id (of the currently selected tab) instead of their own id.

Reply:
Thanks for the explanation!